Dockerfile

```diff
@@ -53,21 +50,10 @@ RUN curl -sfL https://deb.nodesource.com/setup_12.x | bash - && \
 
 FROM js_deps AS python_deps
 
-RUN apt-get install -y locales python3-distutils
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3
-RUN pip3 install -U setuptools cookiecutter
+RUN apt-get install -y locales python3-distutils python3-pip
```

Comment:
Why do we use python here?

Reply:
The `sdcli` is supposed to support golang + python + node projects.

Reply:
> supposed to support
ummm, sort of.  The original intent was python linting, but I don't think we did anything more than that, including anything for js.  Maybe worth putting common lint/format/etc rules in for multiple languages (java *cough)

Reply:
Let's discuss. I am def for chopping up sdcli into more targeted parts/